### ISSUE TITLE
Matrix request returns wrong snapped sources

### ISSUE THREAD

**Issue by arjobsen** [GIScience/openrouteservice#560]
#### Here's what I did
Requesting the duration matrix from 
https:// api.openrouteservice.org /v2/matrix/driving-car
with
{"locations":[[5,52],[6,52],[7,52],[8,52],[9,52]],"destinations":[2,3,4],"sources":[0,1]}

---
#### Here's what I got
```
{"durations":[[7307.56,10348.71,12418.96],[4479.86,7951.64,9704.69]],
"destinations":[{"location":[6.997584,51.999765],"snapped_distance":167.42},{"location":[7.999694,52.000022],"snapped_distance":21.12},{"location":[9.000675,51.998886],"snapped_distance":132.23}],
"sources":[{"location":[6.997584,51.999765],"snapped_distance":167.42},{"location":[7.999694,52.000022],"snapped_distance":21.12},{"location":[9.000675,51.998886],"snapped_distance":132.23}]}
```
The snapped destinations are correct. But the snapped sources are a copy of the destinations.

---
#### Here's what I was expecting
Response with only 2 snapped sources instead of 3 and they should be somewhere near [5,52] and [6,52]

---
#### Here's what I think could be improved
Return the correct locations for the sources

**Comment by arjobsen** [comment]
This also occurs on my local setup with version `'engine': {'version': '5.0.1', 'build_date': '2019-06-24T06:16:58Z'}`

**Comment by rabidllama** [comment]
Hi, thanks for that, the fix will be in the next release. Until then, you can use the older version of the api and that will return the correct items:
`https://api.openrouteservice.org/matrix?profile=driving-car`
with
```json
{
"profile": "driving-car",
"locations":[[5,52],[6,52],[7,52],[8,52],[9,52]],
"destinations":[2,3,4],
"sources":[0,1]
}
```

**Comment by rabidllama** [comment]
Fixed in Release 5.0.2
